### ISSUE TITLE
Wrong detection

### ISSUE THREAD

**Issue by olezhkafp** [ConsoleTVs/Profanity#19]
Hi. for example we have string like this:

`$value = '<p><span style="color: rgb(246,139,106);">Lauren J.</span>, Hi</p>';`

In this case 139 will be replaced as *** because:

Profanity::blocker(trim($value))->badWords() detects the bad word:

```
Array
(
    [2261] => Array
        (
            [language] => zh
            [word] => 13.
        )
)
```

Is it possible to fix? as '139' != '13.'

**Comment by Darthmaul** [comment]
Hey @olezhkafp did you ever figure out a fix for this? We are facing this issue as well

**Comment by ConsoleTVs** [comment]
Hello sorry for the issue. Could we perhaps remove this word ffeom the blocked list? I have no idea about that language but looks odd

**Comment by olezhkafp** [comment]
> 
> 
> Hey @olezhkafp did you ever figure out a fix for this? We are facing this issue as well

@Darthmaul Yes, we care copied the json file with the bad words, modified it and use instead of basic one.

**Comment by olezhkafp** [comment]
> 
> 
> Hello sorry for the issue. Could we perhaps remove this word from the blocked list? I have no idea about that language but looks odd

@ConsoleTVs yes, looks like this is some issue and not the bad word. I think "13." can not be some bad word)

**Comment by ConsoleTVs** [comment]
Hey guys, care to test version `3.0.3` with a potential fix?
